### DISCUSSION TITLE
Add original content type name to roles page when is translated to avoid duplicates

### DISCUSSION THREAD

**Discussion by federicocappellotto97** [directus/directus#9906] [Feature Requests]
I have two content types **posts** and **posts_page**. The second one, since is under **pages** folder, is translated to **posts**
If i go to roles or presets, i found two **posts**, it would be great add the original name if there are translations, like this:

<img width="754" alt="Schermata 2021-11-18 alle 10 54 07" src="https://user-images.githubusercontent.com/63153191/142392580-40ef9d71-98b9-4354-bf06-ccea5210296a.png">

**Comment by rijkvanzanten** [comment]
I think it might make even more sense to simply render the technical name (in monospace) in permissions rather than the prettified name (which would also make it consistent with data-model). Yeehaw @benhaynes?

**Reply by benhaynes** [reply]
Yes. Permissions are technical enough to warrant the actual collection key. Let's keep it with sans-serif font and title format though, and show the actual name (unformatted) and in monospace within a tooltip of the title.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
